tests/units/nulecule/test_nulecule.py

```diff
@@ -0,0 +1,97 @@
+import mock
+import unittest
+from atomicapp.nulecule.base import Nulecule
+
+
+class TestNuleculeRun(unittest.TestCase):
+    """Test Nulecule run"""
+
+    def test_run(self):
+        provider = 'docker'
+        dryrun = False
+        n = Nulecule('some-id', '0.0.2', {}, [], 'some/path')
```

Comment:
it would be useful for readability if these calls used keyword arguments:

```
n = Nulecule(id='some-id', specversion='0.0.2', metadata={}, graph=[], basepath='some/path')
```